integration/tests/session-tasks-sign-in.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, test } from '@playwright/test';
+
+import type { Application } from '../models/application';
+import { appConfigs } from '../presets';
+import type { FakeUser } from '../testUtils';
+import { createTestUtils } from '../testUtils';
+
+test.describe('session tasks sign in flow @nextjs', () => {
+  test.describe.configure({ mode: 'serial' });
+  let app: Application;
+  let fakeUser: FakeUser;
+
+  test.beforeAll(async () => {
+    app = await appConfigs.next.appRouter.clone().commit();
+    await app.setup();
+    await app.withEnv(appConfigs.envs.withSessionTasks);
+    await app.dev();
+
+    const m = createTestUtils({ app });
+    fakeUser = m.services.users.createFakeUser();
+    await m.services.users.createBapiUser(fakeUser);
+  });
+
+  test.afterAll(async () => {
+    await fakeUser.deleteIfExists();
+    await app.teardown();
+  });
+
+  test('navigate to task on after sign-in', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.setIdentifier(fakeUser.email);
+    await u.po.signIn.continue();
+    await u.po.signIn.setPassword(fakeUser.password);
+    await u.po.signIn.continue();
+    await u.po.expect.toBeSignedIn();
+
+    const addOrganizationTask = u.page.getByText(/create organization/i);
+    await expect(addOrganizationTask).toBeVisible();
+    expect(page.url()).toContain('add-organization');
+  });
+
+  test('redirect to task when accessing root sign in component', async ({ page, context }) => {
```

Comment:
Currently, integration tests are failing, it seems that we changed some underlying feature flag logic on FAPI 

I'll also be adding another test suit for the after sign-up flow, covering the same cases regarding redirection